src/js/apps/patients/patient/sidebar/sidebar_app.js

```diff
@@ -25,15 +28,29 @@ export default App.extend({
   },
   onStart({ patient }) {
     this.patient = patient;
+    const currentUser = Radio.request('bootstrap', 'currentUser');
+    this.canManagePatients = currentUser.can('patients:manage');
```

Comment:
When a patient is archived, that patient should be removed from patient search results, correct?

In my local sandbox, the patients are still being included. Not sure if that translates to production, but wanted to make a not of that.

Reply:
I don't know on this one.. I would assume not, but not sure

Reply:
In that scenario, if a user clicks on an archived patient in the search results, the user will hit that 410/404 error page.